sdk/identity/identity/test/manual/src/index.tsx

```diff
@@ -34,12 +37,10 @@ function readClientDetails(): ClientDetails {
 }
 
 function getCredential(clientDetails: ClientDetails): InteractiveBrowserCredential | undefined {
-  return clientDetails.tenantId.length > 0 && clientDetails.clientId.length > 0
```

Comment:
Just to help me follow - are there good reasons to remove the checks?

Reply:
Now that you bring it up, I meant to ask you: is it possible to call InteractiveBrowserCredential without these? because I believe they're optional parameters.

Reply:
I believe so. Nowadays we'd make these required parameters I believe, though this API follows from an early version of Azure SDK.

Reply:
Alright, I'll undo this change.

Reply:
done!